squawkbox/models/baseline.py

```diff
@@ -0,0 +1,64 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence
+
+class Baseline(nn.Module):
+    def __init__(self, vocab_size, embedding_dim, num_lstm_units, num_lstm_layers, padding=0):
+        super().__init__()
+        self.vocab_size = vocab_size
+        self.padding = padding
+
+        self.embedding = nn.Embedding(
+            num_embeddings=vocab_size,
+            embedding_dim=embedding_dim
+        )
+
+        self.lstm = nn.LSTM(
+            input_size=embedding_dim,
+            hidden_size=num_lstm_units,
+            num_layers=num_lstm_layers,
+            batch_first=True
+        )
+
+        self.h2o = nn.Linear(num_lstm_units, vocab_size)
+
+    def forward(self, src, tgt=None, hidden=None):
```

Comment:
Please add `**kwargs`` to the parameters. There are some additional inputs (related to keeping track of time) that will be fed into other models. By adding `**kwargs` you will allow `Baseline` to ignore these other inputs.